### ISSUE TITLE
Screenshot feature: Take a direct picture of the rendered game

### ISSUE THREAD

**Issue by ghost** [hrydgard/ppsspp#19527]
can i request new screenshot feature, i mean the one that just take game screen (like savestate thumbnail did)

**Comment by hrydgard** [comment]
Yeah, having multiple screenshot modes makes sense. Noting it as a feature request.
